packages/slate-editor/src/modules/rich-formatting-menu/RichFormattingMenu.tsx

```diff
@@ -186,9 +194,11 @@ export function RichFormattingMenu({
                 <LinkMenu
                     node={link}
                     canUnlink={link !== null}
+                    withConversionOptions={withConversionOptions}
                     withNewTabOption={withNewTabOption}
                     onBlur={clearLinkRange}
                     onChange={linkSelection}
+                    onConvert={withConversionOptions ? onConvert : undefined}
```

Comment:
`onConvert` will already be `undefined` if `withConversionOptions` is not set (line 178), so this ternary seems redundant 🤔

Reply:
631d1431a1d3703ab84dcfaf50069e5bf207ed0b